Tests/StructuredQueriesTests/EnumTableTests.swift

```diff
@@ -552,17 +392,10 @@
     fileprivate enum Kind {
       case link(URL)
       case note(String)
-      case notes(Attachment.Notes)
-      @Column(as: [String].JSONRepresentation.self)
-      case notesInline([String])
       case video(Attachment.Video)
       case image(Attachment.Image)
     }
 
-    @Selection fileprivate struct Notes {
-      @Column(as: [String].JSONRepresentation.self)
-      let list: [String]
-    }
```

Comment:
@acosmicflamingo Sorry, I reverted these tests since they weren't actually running on CI and were actually failing with the changes I made.

I do wonder if this PR fixes some of the issues you were seeing, though!

Reply:
No big deal! I'll have to see when I'm in front of a computer.